packages/consumption/src/modules/requests/itemProcessors/shareAttribute/ShareAttributeRequestItemProcessor.ts

```diff
@@ -47,7 +48,11 @@ export class ShareAttributeRequestItemProcessor extends GenericRequestItemProces
             }
 
             if (typeof recipient !== "undefined") {
-                const query: any = { "shareInfo.sourceAttribute": requestItem.sourceAttributeId.toString(), "shareInfo.peer": recipient.toString() };
+                const query: any = {
```

Comment:
This adjustment is good, but we still need a similar adjustment for the query of the shared successors and predecessors directly below. (Like you did in the use case `ShareRepositoryAttribute.ts`.)